deploy/aws-ecs/cloudformation.json

```diff
@@ -68,7 +68,7 @@
         },
         "WeaveworksEcsAmiIds": {
             "us-east-1": {
-                "ImageId": "ami-eca289fb"
+                "ImageId": "ami-7b692804"
```

Comment:
lets update the ami for all of the regions to the most update to date weavework ami

Reply:
Done now, also added the regions that were previously missing. All the images from:
https://www.weave.works/docs/scope/latest/ami/